.prettierrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
```

Comment:
Is Single quote, the standard (I read that both single and double quotes can be used as long as the opening quote is the same type as the ending quote).

Reply:
Apparently in our code, we used double quotes mostly. Any reason why single quote is preferred over double quotes? Or is it just a matter of taste?

Reply:
Single quote vs double quotes is just a matter of taste really.
I decided to put single quote since it's usually the preferred one in popular style guides such as [airbnb](https://github.com/airbnb/javascript).